WalletWasabi.Daemon/Global.cs

```diff
@@ -195,6 +195,12 @@ public async Task InitializeNoWalletAsync(TerminateService terminateService, Can
 				{
 					await bitcoinStoreInitTask.ConfigureAwait(false);
 
+					// ToDo: Temporary to fix https://github.com/zkSNACKs/WalletWasabi/pull/12137#issuecomment-1879798750
+					if (AllTransactionStore.MempoolStore.NeedResync || AllTransactionStore.ConfirmedStore.NeedResync)
+					{
+						WalletManager.ResyncToBefore12137();
+					}
```

Comment:
This is the easiest and cleanest way I found to force resync on all wallets from the `TransactionStore`